nms/server/host/routes.ts

```diff
@@ -282,8 +287,59 @@ router.delete(
       });
     });
 
+    await deleteOrc8rTenant(+req.params.id);
     res.status(200).send({success: true});
   }),
 );
 
+async function syncOrganizationWithOrc8rTenant(
+  organization: OrganizationModel,
+): Promise<void> {
+  let orc8rTenant;
+  try {
+    orc8rTenant = (
+      await OrchestratorAPI.tenants.tenantsTenantIdGet({
+        tenantId: organization.id,
+      })
+    ).data;
+  } catch (error) {
+    // Ignore "not found" since there is no guarantee NMS and Orc8r are in sync
+    if (!axios.isAxiosError(error) || error?.response?.status !== 404) {
```

Comment:
Imho the equivalent expression containing only one negation, always feels much simpler to comprehend to me: I.e.
$$\neg ( A \land B) \iff \neg A \lor \neg B$$